src/js/grapl-cdk/lib/historydb.ts

```diff
@@ -36,19 +37,23 @@ export class HistoryDb extends cdk.Construct {
     readonly dynamic_session_table: dynamodb.Table;
     readonly static_mapping_table: dynamodb.Table;
 
-    constructor(scope: cdk.Construct, id: string) {
+    constructor(
+        scope: cdk.Construct,
+        id: string,
+        props: GraplServiceProps,
+    ) {
         super(scope, id);
 
-        this.proc_history = create_table(this, 'process_history_table');
-        this.file_history = create_table(this, 'file_history_table');
-        this.outbound_connection_history = create_table(this, 'outbound_connection_history_table');
-        this.inbound_connection_history = create_table(this, 'inbound_connection_history_table');
-        this.network_connection_history = create_table(this, 'network_connection_history_table');
-        this.ip_connection_history = create_table(this, 'ip_connection_history_table');
-        this.dynamic_session_table = create_table(this, 'dynamic_session_table');
+        this.proc_history = create_table(this, props.prefix + '-process_history_table');
+        this.file_history = create_table(this, props.prefix + '-file_history_table');
+        this.outbound_connection_history = create_table(this, props.prefix + '-outbound_connection_history_table');
+        this.inbound_connection_history = create_table(this, props.prefix + '-inbound_connection_history_table');
+        this.network_connection_history = create_table(this, props.prefix + '-network_connection_history_table');
+        this.ip_connection_history = create_table(this, props.prefix + '-ip_connection_history_table');
+        this.dynamic_session_table = create_table(this, props.prefix + '-dynamic_session_table');
```

Comment:
Can you make sure these table names are correct in the node-identifier?

We should probably just pass in the table names to that service via ENV, which will involve a bit of code change.

Reply:
Oof, yeah looks like that'll break.

Agree about passing the names via env, in the name of the crusade against hard-coded vals. I'll start that.

Reply:
Resolved in newer commits.